### PR TITLE
ci: run tests on WOA hardware

### PR DESCRIPTION
**Pull request by jkleinsc**
#### Description of Change

This PR add WOA testing to our CI by running our tests on Windows on Arm hardware

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: azure-pipelines-woa.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ steps:
   displayName: 'Verify ffmpeg'
 
 - script: |
-    cd src
-    echo "Verifying mksnapshot"
-    python electron\script\verify-mksnapshot.py --build-dir out\Default --source-root %cd%
-  displayName: 'Verify mksnapshot'
+    taskkill /F /IM electron.exe
+    taskkill /F /IM MicrosoftEdge.exe
```
Comment (deepak1556) [inline azure-pipelines-woa.yml]: So the processes don't exit cleanly, is this an issue ? Also why is there a edge process running in ci ?

Comment (brenca) [reply azure-pipelines-woa.yml]: I think the edge process is because of the `shell.openExternal` test where we open a page in the default browser.

Comment (jkleinsc) [reply azure-pipelines-woa.yml]: @deepak1556 those lines are in there to handle bad builds.  The tests don't run in a container, so we need to make sure that the OS is cleaned up for the next test run.   @brenca is right about the edge process running as a result of `shell.openExternal`.

Comment (jkleinsc) [reply azure-pipelines-woa.yml]: @deepak1556 those lines are in there to handle bad builds. The tests don't run in a container, so we need to make sure that the OS is cleaned up for the next test run. @brenca is right about the edge process running as a result of shell.openExternal.

**Pull request merged**